styles/override-editor.scss

```diff
@@ -123,21 +123,68 @@ body {
 		margin: 10px 0;
 	}
 
+	// Bump edit-post-sidebar__panel-tabs z-index.
+	.edit-post-sidebar__panel-tabs {
+		z-index: 2 !important;
+	}
+
 	// Custom responsive field.
-	.custom-responsive {
+	.es-responsive {
+		margin: 0 -1rem;
+		padding: 0 1rem;
+		transition: 0.3s padding ease-out, 0.3s margin ease-out;
+
 		.components-base-control__field {
-			.components-button {
-				margin-left: auto;
+			margin-bottom: 0 !important;
+		}
 
-				svg {
-					margin-right: 0;
-					height: auto;
-					min-width: 100%;
+		> .components-base-control__field {
+			display: flex;
+			flex-direction: column;
+			width: 100%;
+			gap: 0.5rem;
+			padding: 0;
+			margin: 0;
+			transition: 0.3s padding ease-out, 0.3s border-color ease-out;
+			border-radius: 3px;
+			border: 1px solid transparent;
+
+			& > .components-base-control {
+				margin-bottom: 0 !important;
+
+				&:first-of-type {
+					transition: 0.3s padding-left ease-out, 0.3s margin-left ease-out, 0.3s border-left-width ease-out;
+				}
+
+				&:first-of-type:not(.components-base-control__label) {
+					margin-left: 0.75rem;
+					padding-left: 1.25rem;
+					border-left: 1px solid hsl(0, 0%, 94%);
```

Comment:
Why `hsl`?

Reply:
Cause I don't have to worry about calculating (percentage I want * 255)